libs/execution_context/ebpf_native.c

```diff
@@ -377,6 +379,23 @@ ebpf_native_terminate()
     EBPF_RETURN_VOID();
 }
 
+void
+ebpf_object_update_reference_history(void* object, bool acquire, uint32_t file_id, uint32_t line);
+
+static void
```

Comment:
Add comment that this must be called under a lock so the reference history and reference acquisition are atomic?